src/screens/Contact/Contact.tsx

```diff
@@ -35,123 +48,122 @@ const Contact = ({ children }: { children: React.ReactNode }) => {
+          <SectionHeader variant="h4" component="h2" id="join-us">
             Join us!
           </SectionHeader>
-          <Typography variant="body1" component="p">
-            Do you want to join the Covid Act Now team?{' '}
-            <a
-              target="_blank"
-              rel="noopener noreferrer"
-              href="https://docs.google.com/forms/d/e/1FAIpQLSfQkdwXsbDbwLHhWwBD6wzNiw54_0P6A60r8hujP3qnaxxFkA/viewform"
-            >
+          <Paragraph>
+            Covid Act Now is a multidisciplinary team of technologists,
+            epidemiologists, health experts, and public policy leaders working
+            to provide disease intelligence and data analysis on COVID in the
+            U.S. Want to join us?{' '}
+            <ExternalLink href="https://docs.google.com/forms/d/e/1FAIpQLSfQkdwXsbDbwLHhWwBD6wzNiw54_0P6A60r8hujP3qnaxxFkA/viewform">
               Let us know
-            </a>
-            . Right now, we need:
-          </Typography>
+            </ExternalLink>
+            . Some roles that we need:
+          </Paragraph>
           <ul>
-            <li>
-              <Typography variant="body1" component="p">
-                JavaScript/React, Python engineers
-              </Typography>
-            </li>
-            <li>
-              <Typography variant="body1" component="p">
-                Epidemiologists
-              </Typography>
-            </li>
-            <li>
-              <Typography variant="body1" component="p">
-                Data Modelers
-              </Typography>
-            </li>
-            <li>
-              <Typography variant="body1" component="p">
-                ...and more!
-              </Typography>
-            </li>
+            <ListItem>UI/UX Designers</ListItem>
+            <ListItem>JavaScript/React, Python engineers</ListItem>
+            <ListItem>Epidemiologists</ListItem>
+            <ListItem>Data Modelers</ListItem>
+            <ListItem>User Support Representatives</ListItem>
+            <ListItem>Product Managers</ListItem>
+            <ListItem>Social Media Managers</ListItem>
+            <ListItem>...and more!</ListItem>
+          </ul>
+          <Paragraph>
+            Want to intern with us?{' '}
+            <ExternalLink href="https://docs.google.com/forms/d/1rvM5StysnTrhJxXhIdvbTmDvpJKKrfEFl2fQfDt-86Y/viewform?ts=5f248b67&edit_requested=true">
+              Let us know
+            </ExternalLink>
+            . We're currently offering full-time internships (volunteer):
+          </Paragraph>
+          <ul>
+            <ListItem>
+              <ExternalLink href="https://docs.google.com/document/d/1_4P0lM1k2pGRXSNGVe_p8TcmdMdaIug5lZeIK-3zU24/edit?ts=5f2488d3#">
+                Strategy & Operations
+              </ExternalLink>
+            </ListItem>
+            <ListItem>
+              <ExternalLink href="https://docs.google.com/document/d/1HJJ8-W87Z1dGVK1JZF3cDNVyBHSq-Z1MYVoVL4ofZj0/edit#">
```

Comment:
This doc isn't public.

Reply:
good catch - will get this fixed before merging 👍

Reply:
Debbie updated the links, merging!